cli/cli-client/src/main/resources/help/stats.adoc

```diff
@@ -0,0 +1,26 @@
+STATS(1)
+===========
+:doctype: manpage
+
+
+NAME
+----
+stats - displays statistics about resources.
+
+
+SYNOPSIS
+--------
+*stats* ['PATH']
+
+
+EXAMPLES
+--------
+`stats //containers/default` +
+Displays statistics about the default container.
+
+`stats //containers/default/caches/mycache` +
+Displays statistics about the `mycache` cache.
+
+SEE ALSO
```

Comment:
I think it is missing a newline before "SEE ALSO". 
When I run `help stats`, this section is not formatted properly

Reply:
Dibe